### DISCUSSION TITLE
Contributing to Food Oasis.

### DISCUSSION THREAD

**Discussion by victorchrollo14** [hackforla/food-oasis#1693] [General]
I looked up a issue , I wanted to work on , So I tried to fork the project and set it up on my local machine.

Got stuck at 
8) Obtain the dotenv file from the Food-Oasis/Developers G-Drive folder and place it in the /server directory. Re-name this file to .env on your machine. It contains private info (i.e., the production database connection string) that we cannot put in the public GitHub repo.

I was looking for a slack or discord channel to ask this out, But I didn't find any . So I Started the discussion here.

**Comment by qiqicodes** [comment]
Thank you for your interest in contributing to the project and for starting the setup process on your local machine. I'm sorry to hear you've encountered a hiccup along the way.

To join our team, please follow these steps:
- **Start by Onboarding:** Begin your journey by completing the [Hack for LA Onboarding process](https://www.hackforla.org/getting-started). 
- **Joining the Team:** Familiarize yourself with the team's workflows and guidelines by reading the [onboarding section of our Wiki](https://github.com/hackforla/food-oasis/wiki/Joining-the-Team).

We're looking forward to having you on board and navigating this journey together. Should you need further assistance, don't hesitate to reach out!
